### PR TITLE
Created test files for types.go and ledmanagertypes.go

### PR DESCRIPTION
**Pull request by archishou**

### CODE REVIEW

File: pkg/pillar/types/ledmanagertypes_test.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+)
+
+type TestDeriveLedCounterMatrixEntry struct {
+	ledCounter         int
+	usableAddressCount int
+	expectedValue      int
+}
+
+func TestDeriveLedCounter(t *testing.T) {
+	log.Infof("TestLookupIoBundle: START\n")
```
Comment (rvs) [inline pkg/pillar/types/ledmanagertypes_test.go]: Why are these log.Infof here? I see no reason to do this kind of logging.

File: pkg/pillar/types/ledmanagertypes_test.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	testMatrix := []TestDeriveLedCounterMatrixEntry{
+		{ledCounter: 0, usableAddressCount: 0, expectedValue: 1},
```
Comment (rvs) [inline pkg/pillar/types/ledmanagertypes_test.go]: In general, I'd prefer if you could follow the style from this article: https://medium.com/agrea-technogies/basic-testing-patterns-in-go-d8501e360197 when writing Table based testing.

Do you think you can re-write this using that style?

File: pkg/pillar/types/ledmanagertypes_test.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	}
+	for index := range testMatrix {
+		entry := &testMatrix[index]
+		expectedValue := DeriveLedCounter(entry.ledCounter, entry.usableAddressCount)
+		if expectedValue != entry.expectedValue {
+			t.Errorf("Test Entry Index %d Failed: Expected Counter: %d, Actual Counter: %d\n",
+				index, entry.expectedValue, expectedValue)
+		}
+	}
+	log.Infof("TestLookupIoBundle: DONE\n")
```
Comment (rvs) [inline pkg/pillar/types/ledmanagertypes_test.go]: I don't think this should be here

File: pkg/pillar/types/types.go [hunk 4]
```diff
@@ -90,6 +90,18 @@ const (
 	TS_ENABLED
 )
 
+func (e TriState) String() string {
```
Comment (rvs) [inline pkg/pillar/types/types.go]: Why is this required?

File: pkg/pillar/types/types_test.go [hunk 5]
```diff
@@ -0,0 +1,67 @@
+package types
```
Comment (rvs) [inline pkg/pillar/types/types_test.go]: In this file all the comments from a ledmanagertypes_test.go  apply as well.

File: pkg/pillar/types/ledmanagertypes_test.go (hunk 2 above)
Comment (archishou) [reply pkg/pillar/types/ledmanagertypes_test.go]: Would you like me to use the assert package mentioned in the article?

Comment (rvs) [reply pkg/pillar/types/ledmanagertypes_test.go]: testify is a good choice https://github.com/stretchr/testify

**Comment by archishou** [comment]
I resolved the issues you saw with the code previously submitted. Let me know if there is anything else!
